Hackney.Shared.Asset/Domain/AssetCharacteristics.cs

```diff
@@ -7,5 +9,17 @@ public class AssetCharacteristics
         public int NumberOfLivingRooms { get; set; }
         public string WindowType { get; set; }
         public string YearConstructed { get; set; }
+        public int NumberOfBedSpaces { get; set; }
+        public int NumberOfCots { get; set; }
+        public int ShowerRoomAmount { get; set; }
+        public int BathroomAmount { get; set; }
```

Comment:
Will that property be holding information about number of bathrooms? If yes, can it be renamed to 'NumberofBathrooms' for consistency?

Reply:
This has been changed